hrm-domain/hrm-service/permission-rules/nz.json

```diff
@@ -1,27 +1,27 @@
 {
```

Comment:
Are the indentation changes here needed?

Reply:
nope, fixed!